test/dynamo/test_export.py

```diff
@@ -937,22 +937,8 @@ def func(x):
 
         torch._dynamo.reset()
 
-        def compiler(gm, sample_inputs):
-            aten_gm = make_fx(gm)(*sample_inputs)
-
-            self.assertEqual(len(aten_gm.graph.nodes), len(out_graph.graph.nodes))
-            for node1, node2 in zip(aten_gm.graph.nodes, out_graph.graph.nodes):
-                self.assertEqual(node1.op, node2.op)
-                if node1.op == "call_function":
-                    self.assertEqual(node1.target, node2.target)
-                    self.assertEqual(len(node1.args), len(node2.args))
-                    for arg1, arg2 in zip(node1.args, node2.args):
-                        self.assertEqual(type(arg1), type(arg2))
-
-            return aten_gm.forward
-
-        opt_func = torch._dynamo.optimize(compiler, nopython=True)(func)
-        make_fx_result = opt_func(inp)
+        fx_g = make_fx(func)(inp)
+        make_fx_result = fx_g(inp)
```

Comment:
this test seems worse

Reply:
It's better - the graph structure is not a meaningful part of the contract. All that matters is that the answer is the result is the same - what gets lowered to the backend w/ dynamo vs what export produces as its graph is not a contract at all.

Reply:
The original test was the resulting graph was sufficiently normalized that `make_fx`'ing it again was a no-op. That seems like a reasonable invariant to expose.

Reply:
Extended test to cover all 3 ways to get to make_fx.